### DISCUSSION TITLE
drug-drug interaction by protein displacement in plasma in PK-Sim

### DISCUSSION THREAD

**Discussion by mmalsmadi** [Open-Systems-Pharmacology/Forum#912] [How-Tos] [answered]
Hello

Does PK-Sim allow for accounting for drug-drug interaction via displacement of plasma protein binding?
What settings should be done to account for such phenomena in PK-Sim?

**Comment by StephanSchaller** [comment, marked as answer]
Hi @mmalsmadi, this is not a functionality that PK-Sim provides by default but can be achieved by customization through MoBi.
Similar to [this issue](https://github.com/Open-Systems-Pharmacology/Forum/discussions/907), you will have to change fu to be formula-dependent and integrate the DDI therein.

**Comment by mmalsmadi** [comment]
Many thanks, I will try it and keep you posted

Get Outlook for Android<https://aka.ms/AAb9ysg>

________________________________
